exchanges/okx/okx.go

```diff
@@ -2158,30 +2158,24 @@ func (ok *Okx) GetFee(ctx context.Context, feeBuilder *exchange.FeeBuilder) (flo
 	var fee float64
 	switch feeBuilder.FeeType {
 	case exchange.CryptocurrencyTradeFee:
-		var responses []TradeFeeRate
 		uly, err := ok.GetUnderlying(feeBuilder.Pair, asset.Spot)
 		if err != nil {
 			return 0, err
 		}
+		var responses []TradeFeeRate
 		responses, err = ok.GetTradeFee(ctx, okxInstTypeSpot, uly, "")
 		if err != nil {
 			return 0, err
 		} else if len(responses) == 0 {
 			return 0, errors.New("no trade fee response found")
 		}
 		if feeBuilder.IsMaker {
-			if fee, err = strconv.ParseFloat(responses[0].FeeRateMaker, 64); err != nil || fee == 0 {
-				fee, err = strconv.ParseFloat(responses[0].FeeRateMakerUSDT, 64)
-				if err != nil {
-					return fee, err
-				}
+			if fee = responses[0].FeeRateMaker.Float64(); fee == 0 {
+				fee = responses[0].FeeRateMakerUSDT.Float64()
```

Comment:
Please also add in support for USDC based on the supplied quote pair and then this is tACK!